src/components/faqTemplates/ExpandableWithCategories.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { SectionContainer } from '@lib/Containers'
+import { Spacers } from '@lib/Helpers'
+
+const FaqAnswerContainer = styled.div`
+  display: ${props => (props.shouldDisplay ? 'block' : 'none')};
+  overflow: hidden;
+  margin: auto;
+  margin-bottom: 2%;
+`
+
+const Accordion = styled.button`
+  width: 100%;
+  cursor: pointer;
+  text-align: left;
+  outline: none;
+  border: 3px solid black;
+  border-radius: 10px;
+  background-color: transparent;
+`
+
+const FaqQuestionContainer = styled.div``
```

Comment:
is this needed

Reply:
I'm leaving this so I could implement styles when working on High Fi